src/app/static/styles.css

```diff
@@ -347,4 +384,32 @@ table tr:hover {
         display: none;
     }
 }
-/*#endregion */
```

Comment:
My end region! the regions allow us to collapse them in vscode. If we're using a single massive css file rather than a specific one for each page I think we should keep using regions for each page's classes so we can keep em organized and collapse the ones we aren't using. We should probably consider different files for each page if thats possible though. we can still have a common one that is imported in base for common themes but by splitting them we don't have to worry about naming something the same class but on different pages.
Actually thinking about it, is there ever a case where if we felt the need to name it the same class we would want it to look differently?

Reply:
I did not realize what it did. As for different files we could do that, but as you said getting them imported gets more complicated. I also dont see a reason why we would use a same name but want a different look.